docs/ui.md

```diff
@@ -27,6 +27,56 @@ These elements are defined in the manifest.xml file:
 </connector-plugin>
 ```
 
+## Vendor Attributes (Custom Fields)
+Vendors can add customized attributes to their connector plugin by using the vendor attributes.
+
+These fields have a custom label and can be used for attributes in the connection strings that are not available in the attribute list. You can currently add 3 custom fields in your connector plugin.
+
+To add a custom vendor-attribute, you will need to modify your connection-dialog.tcd, connectionResolver.tdr and connectionProporties.js file like any other attribute.
```

Comment:
connectionBuilder or connectionProperties. The properties builder script is only for jdbc connectors, for odbc connectors you will be adding these to the connection builder script.

Reply:
Primary documentation should probably be to use them in connectionBuilder.js.  Optionally in jdbc they could also be used in a propertiesBuilder.js